demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -134,22 +182,49 @@ class ContentList extends React.Component {
   changeTransportType(transportType) {
     const contents = CONTENTS_PER_TYPE[transportType];
     const firstEnabledContentIndex =
-      contents.findIndex((content) => !content.disabled);
+      contents.findIndex((content) => !content.disabled && content.name !== "Custom link");
     this.setState({
       transportType,
       contentChoiceIndex: firstEnabledContentIndex,
-      hasTextInput: CONTENTS_PER_TYPE[transportType].length - 1 ===
-        firstEnabledContentIndex,
+      hasTextInput: firstEnabledContentIndex === 0,
+      manifestUrl: "",
+      contentName: "",
+      licenseServerUrl: "",
+      setServerCertificate: "",
+      isSavingOrUpdating: false,
+      content: undefined,
     });
   }
 
-  changeContentIndex(index) {
-    const { transportType } = this.state;
-    const hasTextInput = CONTENTS_PER_TYPE[transportType].length - 1 === index;
+  changeContentIndex(index, content) {
+    const hasTextInput = index === 0;
+
+    let manifestUrl = "";
+    let contentName = "";
+    let licenseServerUrl = "";
+    let setServerCertificate = "";
+
+    if (content) {
+      const { localContent } = content;
+      manifestUrl = localContent ? content.url : "";
+      contentName = localContent ? content.name : "";
+      licenseServerUrl = (localContent && content.drmInfos && content.drmInfos[0]) ?
+        content.drmInfos[0].licenseServerUrl :
+        "";
+      setServerCertificate = (localContent && content.drmInfos && content.drmInfos[0]) ?
+        content.drmInfos[0].setServerCertificate :
+        "";
```

Comment:
So if ``localContent`` is false we stay with ``""`` for every fields:
  1. why
  2. wouldn't that be better to just update the `if instruction` instead